wnfs/src/common/link.rs

```diff
@@ -215,37 +224,99 @@ where
     }
 }
 
+pub trait RemembersPersistence {
```

Comment:
Nit: Thinking there should be a mention of Cid in the name because that is what it is really about. Maybe like `CidPersistable`, `PersistAsCid`, etc.

Reply:
How about `RemembersCid`?

Reply:
Sounds fine